.github/workflows/cleanup-webapp-test.yaml

```diff
@@ -14,11 +14,22 @@ on:
     - 'lib'
 
 jobs:
+  unit:
+    name: Unit Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version-file: .nvmrc
+    - run: npm ci
```

Comment:
We can do `npm ci --workspace <workspace_name>` to avoid installing all the dependencies.

Reply:
Hm, it works fine in https://github.com/brandwatch/frontend-apps/blob/main/platform/importmap-service/Dockerfile#L6 :thinking:

Reply:
Seems like the workflow is using  root dependencies eg. `eslint`

https://github.com/BrandwatchLtd/slipstream-actions/actions/runs/3966534065

Reply:
I am reverting. These workflows run that rarely I don't think we need to optimise them 🙂